cypress/features/regression/test/actionPopoverNoIFrame.feature

```diff
@@ -103,14 +104,14 @@ Feature: Action Popover component in noIFrame
     Then Action Popover element is not visible
 
   @positive
-  Scenario: Close ActionPopover using ESC key
+  Scenario: Close ActionPopover using ESC key if it hasn't a submenu
```

Comment:
you still don't check if the focused element has submenu or not

Reply:
<img width="923" alt="Screenshot 2020-03-24 at 08 49 55" src="https://user-images.githubusercontent.com/34001198/77401048-84eabc00-6dac-11ea-8b2f-35684e22b942.png">